updated Premium storage disks - include Esv3-series

A139064 opened the pull request:
Please note a per this page:
https://docs.microsoft.com/en-au/azure/virtual-machines/windows/sizes-memory#esv3-series-sup1sup

Esv3 uses premium storage

azurecla commented:
Hi __@ppatierno__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@ppatierno__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

djrosanova commented:
Approved. Great addition!

ppatierno commented:
Great ! Thanks ;-)

A139064 committed:
updated Premium storage disks - include Esv3-series

Please note a per this page:
https://docs.microsoft.com/en-au/azure/virtual-machines/windows/sizes-memory#esv3-series-sup1sup

Esv3 uses premium storage
